vllm/platforms/cuda.py

```diff
@@ -195,9 +195,16 @@ def get_attn_backend_cls(cls, selected_backend, head_size, dtype,
                 logger.info("Using Triton MLA backend.")
                 return "vllm.attention.backends.triton_mla.TritonMLABackend"
         if use_v1:
-            logger.info_once("Using Flash Attention backend on V1 engine.")
-            return ("vllm.v1.attention.backends.flash_attn."
-                    "FlashAttentionBackend")
+            if cls.has_device_capability(80):
```

Comment:
I think we should remove these logs

Reply:
Why should we? I think people care about this log (although I really want to provide only one option per hardware).